introspection.go

```diff
@@ -145,15 +146,23 @@ func (p *Peer) IntrospectState(opts *IntrospectionOptions) PeerRuntimeState {
 	p.mut.RLock()
 
 	hostPort := p.hostPort
-	conns := make([]ConnectionRuntimeState, len(p.connections))
-	for i, conn := range p.connections {
-		conns[i] = conn.IntrospectState(opts)
+	inboundConns := make([]ConnectionRuntimeState, len(p.inboundConnections))
```

Comment:
I updated all other nits except this one. I don't think it is necessary. The exist code just one line and this is only place to use it. I would keep it as it is.